controller/idgen/id.go

```diff
@@ -51,10 +51,20 @@ func (self *shortIdGenerator) NextId() (string, error) {
 	}
 }
 
-func NewUUIDString() string {
+func MustNewUUIDString() string {
```

Comment:
"MustNew" ?

Reply:
I would guess that the error never happens, but before we were ignoring the error. I figured it was better to find out that something had gone wrong than not.